### DISCUSSION TITLE
Banner Github Star

### DISCUSSION THREAD

**Discussion by bryanchappell** [hyper63/atlas#92] [Ideas] [answered]
Addition of a banner at the top of the site recommending visitors to star the Hyper63 repo. Example screenshot below:

<img width="587" alt="Screen Shot 2021-01-31 at 3 31 11 PM" src="https://user-images.githubusercontent.com/6373486/106397052-85df7780-63d9-11eb-8126-504e59166c0e.png">

Text would be hyperlinked to open the repo in a new tab. [This](https://tailwindui.com/components/application-ui/overlays/notifications#component-a065edf8f08a35f06943b2e266ca9af1) notification from TailwindUI might work well.

**Comment by twilson63** [comment, marked as answer]
Adding issue to website repository!
